src/software/ai/hl/stp/play/penalty_kick_play.cpp

```diff
@@ -14,44 +14,61 @@ std::string PenaltyKickPlay::getName() const
 
 bool PenaltyKickPlay::isApplicable(const World &world) const
 {
-    if (world.gameState().isOurPenalty())
-    {
-        return true;
-    }
-    else
-    {
-        return false;
-    }
+    return (world.gameState().isReadyState() || world.gameState().isSetupState()) &&
+           world.gameState().isOurPenalty();
```

Comment:
Nit: Brackets around the && statement please

Reply:
fixed

Reply:
whoops I misread that

Reply:
my bad